src/plugins/htmlAudioPlayer/plugin.js

```diff
@@ -112,9 +112,14 @@ class HtmlAudioPlayer {
             let val = options.url;
             console.debug('playing url: ' + val);
             import('../../scripts/settings/userSettings').then((userSettings) => {
-                if (userSettings.enableAudioNormalization() && options.item.LUFS != null) {
+                if (userSettings.selectAudioNormalization() == 'TrackGain' && options.item.LUFS != null) {
                     const dbGain = -18 - options.item.LUFS;
                     self.gainNode.gain.value = Math.pow(10, (dbGain / 20));
+                    console.debug('Using track gain');
+                } else if (userSettings.selectAudioNormalization() == 'AlbumGain' && options.mediaSource.albumLUFS != null) {
+                    const dbGain = -18 - options.mediaSource.albumLUFS;
+                    self.gainNode.gain.value = Math.pow(10, (dbGain / 20));
+                    console.debug('Using album gain');
```

Comment:
Same here.

Reply:
I have added/updated these will the latest push